doc/.readthedocs.yaml

```diff
@@ -22,8 +29,8 @@ sphinx:
   fail_on_warning: true
 
 # If using Sphinx, optionally build your docs in additional formats such as PDF
-formats:
-   - pdf
+#formats:
+#   - pdf
```

Comment:
Are pdf builds just not being used/exposed?

Reply:
This disables PDF builds.

Since we are combining the documentation, the default PDF build probably won't give us a useful result.  (I mean, the [existing build](https://canonical-microcloud.readthedocs-hosted.com/_/downloads/en/latest/pdf/) doesn't either ... :wink: )
It should be feasible to get this to work, but it'll need some looking into.